History.md

```diff
@@ -1,4 +1,19 @@
-# Changelog
+## 1.29.0 / 2024-12-11
+
+* Default to "tar.xz" archive file format for Linux (#709)
+* Remove support for Python 3.8 (#706)
+* Add workflow to auto publish releases on PyPI (#686)
+* Fix remote tests (#692)
+* Use httpd fixture per session to increase test runtime drastically (#690)
```

Comment:
nit: I guess you meant either "decrease" or "improve", increasing test runtime sounds like an issue/regression

Reply:
Oh yes! Good spot! Thanks.